app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -102,6 +102,14 @@ data class Patient(
     @Query("DELETE FROM patient")
     abstract fun clear()
 
+    @Query("""
+      SELECT P.* FROM Patient P
+        INNER JOIN BusinessId B ON B.patientUuid == P.uuid
+        WHERE B.identifier == :identifier AND B.deletedAt IS NULL
+        ORDER BY B.createdAt ASC
```

Comment:
Can you align these last three lines with the first one?

Reply:
Done.